content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -19,11 +17,11 @@ The application has the following components:
 
 ![architecture diagram](https://res.cloudinary.com/prismaio/image/upload/v1628765080/docs/ybRH5wH_bpimat.png)
 
-The focus of this guide is showing how Prisma integrates with Netlify. The starting point will the [Prisma Netlify example](https://github.com/prisma/deployment-example-netlify) which has a couple of REST endpoints preconfigured as serverless functions and a static HTML page.
+The focus of this guide is to show how Prisma integrates with Netlify. The starting point is the [Prisma Netlify example](https://github.com/prisma/deployment-example-netlify) which has a couple of REST endpoints preconfigured as serverless functions and a static HTML page.
 
 With Netlify, the fundamental building block is a [**Site**](https://docs.netlify.com/configure-builds/get-started/). Sites are typically connected to a Git repository and have a publicly accessible URL that looks like `https://site-name.netlify.com`. In this guide, you will connect a GitHub repository to a Netlify site.
```

Comment:
```suggestion
With Netlify, the fundamental building block is a [**Site**](https://docs.netlify.com/configure-builds/get-started/). Sites are typically connected to a Git repository and have a publicly accessible URL that looks like `https://site-name.netlify.app`. In this guide, you will connect a GitHub repository to a Netlify site.
```
The domain extension changed at some point, check if it's mentioned somewhere else just in case

Reply:
That's a great catch. I never understood why `curl https://<site_name>.netlify.com` in the later stages of the guide returned `Moved permanently'. It'll now be fixed. Thanks, @Jolg42.